.changeset/modern-scissors-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-development-kitchen': major
+---
+
+Make ticker size configurable
```

Comment:
You could add some of the information about renaming properties and the new size argument